OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -383,12 +383,15 @@ public bool IsVisible(Actor a)
 
 		public bool IsTargetable(Actor a)
 		{
-			if (a.TraitsImplementing<IVisibilityModifier>().Any(t => !t.IsVisible(a, self.Owner)))
+			if (a.TraitsImplementing<IStealth>().Any(t => !t.IsVisible(a, self.Owner)))
```

Comment:
It shouldn't be. You'll need to think of another way to do that.

On 2015-05-18, at 8:45 PM, Deniz <notifications@github.com> wrote:

> In OpenRA.Game/Traits/World/Shroud.cs:
> 
> > @@ -383,12 +383,15 @@ public bool IsVisible(Actor a)
> >  
> >  		public bool IsTargetable(Actor a)
> >  		{
> > -			if (a.TraitsImplementing<IVisibilityModifier>().Any(t => !t.IsVisible(a, self.Owner)))
> > +			if (a.TraitsImplementing<IStealth>().Any(t => !t.IsVisible(a, self.Owner)))
> OK how should the mod independent interface be called then if stealth is mod specific?
> 
> ―
> Reply to this email directly or view it on GitHub.
>